test/Microsoft.SqlTools.ManagedBatchParser.IntegrationTests/BatchParser/BatchParserTests.cs

```diff
@@ -338,6 +338,42 @@ public void VerifyIsSqlCmd()
             }
         }
 
+        /// <summary>
+        /// Verify whether the batchParser execute SqlCmd successfully
+        /// </summary>
+        [Fact]
+        public void VerifyRunSqlCmd()
+        {
+            using (ExecutionEngine executionEngine = new ExecutionEngine())
+            {
+                string sqlCmdQuery = @"
+:setvar __var1 1
```

Comment:
That would be your call. There is no downside/risk to adding the keyword const just a very miniscule perf gain. Ideally all our code with literal strings would attributed as 'const' and if they use runtime values be attributed readonly but I do not have the luxury of making that change across the board. What would be nice if we start doing it right new code and change code as we come across it.

Reply:
will do